### PR TITLE
feat: add multiarch debian docker image

### PR DESCRIPTION
**Pull request by soulbird**

### CODE REVIEW

File: debian/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        && echo "deb http://repos.apiseven.com/packages/debian $codename main" | tee /etc/apt/sources.list.d/apisix.list \
+        ;; \
+      arm64) \
+        echo "deb http://openresty.org/package/arm64/debian $codename openresty" | tee /etc/apt/sources.list.d/openresty.list \
+        && wget -O - http://repos.apiseven.com/pubkey.gpg | apt-key add - \
+        && echo "deb http://repos.apiseven.com/packages/arm64/debian $codename main" | tee /etc/apt/sources.list.d/apisix.list \
+        ;; \
+    esac; \
+    apt update \
+    && apt install -y apisix=${APISIX_VERSION}-0 \
```
Comment (spacewander) [inline debian/Dockerfile:44]:
```suggestion
    && apt install -y apisix-${APISIX_VERSION}-0 \
```
Typo?

File: debian/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    && openresty -V \
+    && apisix version
```
Comment (spacewander) [inline debian/Dockerfile:46]: Do we need to clean up apt? Like:
https://github.com/nginxinc/docker-nginx/blob/f3d86e99ba2db5d9918ede7b094fcad7b9128cd8/Dockerfile-debian.template#L85

File: debian/Dockerfile (hunk 1 above)
Comment (soulbird) [reply debian/Dockerfile:44]: This means to install the specified APISIX version.

**Comment by soulbird** [comment]
> Did it run successfully in your own CI?

https://hub.docker.com/layers/apisix/soulbird/apisix/2.14.1-mutil/images/sha256-37d0ec18d8d0bec0f2a9c70775e79b9b687639556605c7563b2b460aebea34ec?context=repo&tab=layers

**Review by bzp2010 — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**